pulp_service/pulp_service/app/tasks/package_scan.py

```diff
@@ -14,13 +14,20 @@
 content_queue = Queue()
 
 
-async def check_content(repo_version_pk):
+async def check_content(repo_version_pk=None, npm_package=None):
```

Comment:
I know this is from the other PR, but I'd probably rename this to `check_repo_version` and leave this function to only check the repo version. Saving the few lines of code is not worth mixing the two calls together with an if/else. It increases the cyclomatic complexity and decreases the separation of concerns. You can go read about those topics if they are new. That's the motivation I offer in making a whole new function that does repeat some code.